Recipes/Recipes.Application/DTOs/Recipes/RecipeCreationDTO.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
```

Comment:
Not a big issue, but I think many of these imports are unused and not only in this particular class